II.pass-tests/10-paint-ford/paint-cars.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function paintShop(cars, color) {
+  var newCars = cars.map(car => {
```

Comment:
Could you rewrite this using the ES6 object destructuring pattern you've used in the other exercise?